flock_agent/gui/onboarding.py

```diff
@@ -284,13 +284,23 @@ def __init__(self, common):
         label.setWordWrap(True)
         label.setStyleSheet(self.c.gui.css["Onboarding label"])
 
-        # TODO: make separate macOS/Linux screenshots for this icon
         systray_image = QtWidgets.QLabel()
-        systray_image.setPixmap(
-            QtGui.QPixmap.fromImage(
-                QtGui.QImage(self.c.get_resource_path("images/onboarding-systray.png"))
+        if Platform.current() == Platform.MACOS:
+            systray_image.setPixmap(
+                QtGui.QPixmap.fromImage(
+                    QtGui.QImage(
+                        self.c.get_resource_path("images/onboarding-systray.png")
```

Comment:
What about assigning the image path to a variable in the if/else and the using the variable outside of this. Mostly because I stared at this for an embarrassingly long time trying to figure out what was different between the two.